include/deal.II/matrix_free/matrix_free.h

```diff
@@ -640,6 +640,13 @@ class MatrixFree : public Subscriptor
    */
   //@{
   /**
+   * Returns whether a given FiniteElement @p fe is supported by this class.
+   */
+  template <int spacedim>
+  static
+  bool supports_fe (const FiniteElement<dim, spacedim> &fe);
```

Comment:
that is: MatrixFree data supports a given FE instead of MatrixFree data is supported by a given FE